tools/telemetry-generator/src/handlers/testPassRate.ts

```diff
@@ -11,7 +11,7 @@
  * @param logger - An ITelemetryBufferedLogger. Call its send() method to write the output telemetry events.
  */
 module.exports = function handler(fileData, logger): void {
-	// eslint-disable-next-line @rushstack/no-new-null -- External API can return an actual null
+	 
```

Comment:
This is an important note about the external API. Plus I expect this is going to complain without the rule disable, due to us using `null` which our lint rules don't like.

Reply:
I ran lint:fix and it got removed. Put it back